src/wallet/asyncrpcoperation_mergetoaddress.cpp

```diff
@@ -23,16 +25,15 @@
 #include <wallet/wallet.h>
 #include <wallet/walletdb.h>
 #include <zcash/IncrementalMerkleTree.hpp>
+#include <key_io.h>
 
 #include <chrono>
 #include <iostream>
 #include <string>
 #include <thread>
 
-#include <wallet/paymentdisclosuredb.h>
-
 using namespace libzcash;
-
+#if 0
```

Comment:
better to just remove this file from compilation right now instead of ifdefing a bunch of it out?
then mergetoaddress can be a separate pr?